### PR TITLE
declutter the history 'cog' menu

### PR DESCRIPTION
**Pull request by martenson**
- [x] move the 'import from file' to the saved histories page

### CODE REVIEW

**Comment by dannon** [comment]
@martenson Looks like a reasonable first step forward -- I'm not sure how to best approach most of the other stuff.  Want to just get this merged and take the rest of it in separate PRs?

**Comment by martenson** [comment]
ready for review

File: client/galaxy/scripts/mvc/history/options-menu.js [hunk 1]
```diff
@@ -172,16 +172,6 @@ var menu = [
...
         href: "history/export_archive?preview=True",
         anon: true
-    },
-
-    {
-        html: _l("Other Actions"),
-        header: true
-    },
-    {
-        html: _l("Import from File"),
```
Comment (nsoranzo) [inline client/galaxy/scripts/mvc/history/options-menu.js:212]: What about putting it together with the previous section, to be renamed from `Downloads` to `Import/Export` ?

Comment (martenson) [reply client/galaxy/scripts/mvc/history/options-menu.js:212]: I still think it does not belong in the current history 'context' menu at all.

Comment (nsoranzo) [reply client/galaxy/scripts/mvc/history/options-menu.js:212]: We have "Create New" under the "Current History" header...

Comment (martenson) [reply client/galaxy/scripts/mvc/history/options-menu.js:212]: That should imo go away too, along with other options.

Comment (nsoranzo) [reply client/galaxy/scripts/mvc/history/options-menu.js:212]: Thanks for #7565 ! IMO import and export should be together, but it's a -0.

**Review by jmchilton — APPROVED** [review]

**Comment by martenson** [comment]
thank you @nsoranzo and @jmchilton

**Pull request merged**